components/Microcopy/Microcopy.tsx

```diff
@@ -0,0 +1,50 @@
+import { FunctionComponent } from "react";
+import { TypeMicrocopy } from "../../generated/contentful";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTriangleExclamation } from "@fortawesome/free-solid-svg-icons";
+import Style from "./styles.module.scss";
+
+export interface MicrocopyProps {
+  entries: TypeMicrocopy[];
+  id: string;
+}
+
+const Microcopy: FunctionComponent<MicrocopyProps> = ({ entries, id }) => {
+  let microcopy = entries.find((entry) => entry.sys.id == id);
+
+  if (
+    typeof window !== "undefined" &&
+    window.sessionStorage.getItem("editingMode")
+  ) {
+    // client side
+    return (
+      <span
+        className={Style.editing}
+        onClick={() => {
+          window.open(
+            `https://app.contentful.com/spaces/dtb5w0ega2aw/entries/${id}`,
```

Comment:
oh that space id is an env var, you should get it from there instead

Reply:
The environment variables are only available server side, but this code runs on the client side.

Reply:
yes, however they can be exposed: 
https://nextjs.org/docs/basic-features/environment-variables

Reply:
i still think you should change this so the id is just in one place